ooniapi/services/oonifindings/src/oonifindings/routers/v1.py

```diff
@@ -3,26 +3,26 @@
 """
 
 from datetime import datetime, timezone
-from typing import List, Dict, Optional, Union, Tuple, Any, Annotated
```

Comment:
Should we maybe bump the version number on the router endpoint? As in make into `v2`. That way if we break something badly we can always revert back to speaking to the old backend as `v1`